image/postgres/konflux.Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM registry.redhat.io/rhel8/postgresql-13:latest AS final
+FROM registry.redhat.io/rhel8/postgresql-15:latest AS final
```

Comment:
nit:  For consistency would it be worth doing it with `ARG PG_VERSION=15` here as well?